parser_library/src/diagnostic.cpp

```diff
@@ -2097,6 +2097,14 @@ diagnostic_op diagnostic_op::error_E077(const range& range)
     return diagnostic_op(diagnostic_severity::error, "E077", "Statement count limit reached", range);
 }
 
+diagnostic_op diagnostic_op::error_E078(std::string_view message, const range& range)
+{
+    return diagnostic_op(diagnostic_severity::error,
+        "E078",
+        concat("Inconsistent GBL symbol definitions, first is retained - ", message),
```

Comment:
GBL -> global variable?
The message is a bit mysterious, I am not sure I would immediately be able to tell what the problem is.

maybe `Global variable re-declared with an incorrect type...`?